lmcache/v1/storage_backend/local_cpu_backend.py

```diff
@@ -244,19 +244,13 @@ def unpin(self, key: CacheEngineKey) -> bool:
             return True
 
     def remove(self, key: CacheEngineKey, force: bool = True) -> bool:
-        if force:
-            self.cpu_lock.acquire()
-        if key not in self.hot_cache:
+        with self.cpu_lock:
```

Comment:
Good point, But need to make sure that it is not affecting to the layerwise mode because it seems previously, we can remove the object w/o locking. @YaoJiayi, Could you confirm that this changes? it seems coming from your refactor before (https://github.com/LMCache/LMCache/commit/451501601d79181ab5d177bff08949f7459b39d3) and previously it is similar like this changes.

And maybe it serializes the async behavior with multi-core and high concurrency requests.